spec/features/deposit_via_xlsx_spec.rb

```diff
@@ -1,13 +1,31 @@
 # frozen_string_literal: true
 
+require 'rubyXL'
+require 'rubyXL/convenience_methods/cell'
+
 RSpec.describe 'Use Argo to upload metadata in a spreadsheet', type: :feature do
-  let(:start_url) { 'https://argo-stage.stanford.edu/view/druid:qc410yz8746' }
+  let(:argo_home) { 'https://argo-stage.stanford.edu/' }
+  let(:start_url) { "#{argo_home}view/druid:qc410yz8746" }
+  let(:druid1_url) { "#{argo_home}view/fh141gk9610" }
```

Comment:
How are these druids created?

Reply:
These druids were manually created to match spreadsheet fixture.

Reply:
Is it possible that we register new druids just for this test?  This will greatly reduce the overhead of setting up this test script when we clear out the staging environment.
If we can't we need to document how they need to be made so that we can initialize these fixtures so that the tests will work.

Reply:
Done. Added new spec helper module to register the new druids.